app/experimenter/nimbus-ui/src/components/PageEditMetrics/index.tsx

```diff
@@ -28,23 +28,23 @@ const PageEditMetrics: React.FunctionComponent<RouteComponentProps> = () => {
 
   const onSave = useCallback(
     async ({
-      primaryProbeSetIds,
-      secondaryProbeSetIds,
-    }: Record<string, number[]>) => {
+      primaryProbeSetSlugs,
+      secondaryProbeSetSlugs,
+    }: Record<string, string[]>) => {
       try {
         const nimbusExperimentId = currentExperiment.current!.id;
         const result = await updateExperimentProbeSets({
           variables: {
             input: {
               id: nimbusExperimentId,
-              primaryProbeSetIds,
-              secondaryProbeSetIds,
+              primaryProbeSetSlugs,
+              secondaryProbeSetSlugs,
             },
           },
         });
 
         if (!result.data?.updateExperiment) {
-          throw new Error("Save failed, no error available");
+          throw new Error(SUBMIT_ERROR);
```

Comment:
Good catch 👍

Reply:
Now you done got me curious... Here's the other places where we seemingly throw new Errors (some of which with hardcoded strings, some not; potentially none relevant):

```sh
git grep -n "throw new " app/experimenter/nimbus-ui

app/experimenter/nimbus-ui/README.md:158:    throw new Error("Update failed for an unknown reason");
app/experimenter/nimbus-ui/src/components/AppErrorBoundary/index.test.tsx:42:      throw new Error("bad");
app/experimenter/nimbus-ui/src/components/PageEditAudience/index.tsx:57:          throw new Error(SUBMIT_ERROR);
app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/index.test.tsx:110:      throw new Error(expectedErrorMessage);
app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/reducer/update.ts:27:    throw new UpdateStateError("Control branch is required");
app/experimenter/nimbus-ui/src/components/PageEditBranches/index.tsx:58:          throw new Error(SUBMIT_ERROR_MESSAGE);
app/experimenter/nimbus-ui/src/components/PageEditMetrics/index.tsx:47:          throw new Error("Save failed, no error available");
app/experimenter/nimbus-ui/src/components/PageEditOverview/index.tsx:53:          throw new Error("Save failed, no error available");
app/experimenter/nimbus-ui/src/components/PageNew/index.tsx:43:          throw new Error("Save failed, no error available");
app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx:97:        throw new Error(SUBMIT_ERROR);
app/experimenter/nimbus-ui/src/index.test.tsx:71:      throw new Error("uh oh");
app/experimenter/nimbus-ui/src/services/config.ts:30:      throw new Error("Configuration is empty");
app/experimenter/nimbus-ui/src/services/config.ts:42:      throw new Error(`Invalid configuration ${JSON.stringify(content)}: ${decoded}`,
```

Reply:
@pdehaan I noticed that too. We have this string hanging out in a few places as well as called `SUBMIT_ERROR_MESSAGE` in another file. I'll create a follow up (good maintenance item!) to move them in our `constants` file. :)

Reply:
Issue here: #4597